include/evmmax/evmmax.hpp

```diff
@@ -8,20 +8,23 @@
 namespace evmmax
 {
 
-/// Compute the modulus inverse for Montgomery multiplication, i.e. N': mod⋅N' = 2⁶⁴-1.
+/// Compute the modulus inverse for Montgomery multiplication, i.e., N': mod⋅N' = 2⁶⁴-1.
```

Comment:
I would change `mod` -> `M`. Now it's confusing with `K mod N'`

Reply:
But isn't the modulus named `N`?